packages/postgres-database/src/simcore_postgres_database/utils_aiosqlalchemy.py

```diff
@@ -8,7 +8,7 @@
 
 async def get_pg_engine_stateinfo(engine: AsyncEngine) -> dict[str, Any]:
     return {
-        "current pool connections": f"{engine.pool.checkedin()=},{engine.pool.checkedout()=}",
+        "current pool connections": f"{engine.pool.checkedin()=},{engine.pool.checkedout()=}",  # type: ignore
```

Comment:
maybe you can remote the type ignore if you return a `dict[str, str]`?

Reply:
that is not the problem. The problem is that it cannot find ``checkedin`` in the pool but it actukally exists